modules/ROOT/pages/contribute.adoc

```diff
@@ -7,15 +7,12 @@ Many of our documentation repositories are open and we welcome your contribution
 
 To contribute to the English version of this documentation:
 
-. Select *Edit in Github* on the page you want to modify.
+. If this is your first time contributing, sign the https://cla.salesforce.com/sign-cla[Salesforce.com Contributor License Agreement (CLA)]. The CLA only needs to be signed once per GitHub account.
+. Click *Edit in Github* on the page you want to modify.
```

Comment:
Because I am guessing that there is user navigation involved between the first step and this, I suggest locating the user first:
```suggestion
. On the page you want to modify, click *Edit in Github*.
```
OR
```suggestion
. Navigate to the page that you want to modifiy and click *Edit in Github*.
```

Reply:
By the way, we have not talked about this yet, but you own your work. Period.

If I make a grammar or style or copy edit, you should definitely implement those (if you think you have an exception, and there are some, please ask). 

The rewordings that I suggest are for you to evaluate based on your own knowledge of your intent and technical precision. 

The questions that I ask are not for you to answer for me, but for your reader: please similarly consider them from the audience point of view and adjust text as you see fit.

We can discuss this further if you like!